navigation-material/src/main/java/com/google/accompanist/navigation/material/SheetContentHost.kt

```diff
@@ -141,11 +139,7 @@ internal fun SheetContentHost(
             onDispose {
                 scope.launch {
                     hideCalled = true
-                    try {
-                        sheetState.internalHide()
-                    } finally {
-                        hideCalled = false
-                    }
+                    sheetState.internalHide()
```

Comment:
Certainly not the ideal solution but if it works I can come back to it next week :)